src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -225,7 +225,7 @@
     </IntProperty.DataSource>
   </IntProperty>
 
-  <StringProperty Name="PackageIconUrl"
+  <StringProperty Name="PackageIcon"
```

Comment:
@dominofire can you confirm the above?

Reply:
@karann-msft @dominoFire is there anything blocking this?

Reply:
@etbyrd , @davidwengier We only warn if there's `<PackageIconUrl/>` present in the project file. When you add an `<PackageIcon/>`, NuGet will not warn at pack, even if both properties are present.

The NuGet Pack targets get all the properties from the project file. As long as the Xaml only modifies `<PackageIcon/>`, I think we're good.

Regarding the warning message for licenses (#4223), it would be a nice-to-have, for keeping consistency between Embedded Licenses and Embedded Icons.

Reply:
@dmonroym - we should address the above feedback from @dominofire

Reply:
Discussed offline with @melytc . The warning message was added, thanks! We should follow-up if whether or not remove `<PackageIconUrl/>` from the project file when the Icon is added/updated/deleted from the Project Properties dialog